target/linux/ramips/dts/MIR3P.dts

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "xiaomi,mir3p", "mediatek,mt7621-soc";
+	model = "Xiaomi Mi Router 3 Pro";
+
+	aliases {
+		led-boot = &led_status_blue;
+		led-failsafe = &led_status_blue;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_blue;
```

Comment:
Just curious: 

Is there a section in the device's manual that describe the function of the leds in detail? Because it's a definitely plus, if the assigned LED function match does from the vendor firmware.

Reply:
Very good suggestion (I had just copy/pasted this code from MIR3G).

The manual (http://files.xiaomi-mi.co.uk/files/router_pro/router%20PRO%20EN.pdf) page 5 says the following about indicator lights:

yellow: "updated" or "started"
blue: normal functioning
red: "safe mode" (or "system failure")

Given that, I think the following would make sense:

```suggestion
		led-boot = &led_status_yellow;
		led-failsafe = &led_status_red;
		led-running = &led_status_blue;
		led-upgrade = &led_status_yellow;
```
Does that seem about right?

Reply:
> Does that seem about right?

:+1: